src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/Example2.java

```diff
@@ -0,0 +1,24 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MatchXpath">
+      <property name="query"
+       value="//CTOR_DEF[count(./PARAMETERS/*) > 0]"/>
+      <message key="matchxpath.match"
+      value="Parameterized constructors are not allowed,
+      there should be only default ctor"/>
```

Comment:
Same is here

Reply:
We can make `value="Parameterized constructors are not allowed"`.
Second part is over descriptive , we can avoid it in example

Reply:
Done.